js/components/discussions/thread-form.vue

```diff
@@ -11,47 +11,51 @@
 </template>
 
 <script>
-import log from 'logger';
+    import log from 'logger';
```

Comment:
Can you revert the indentation as before? I can't review the changes ;-)

Reply:
Done :)